src/components/xcpl_comps_nuopc/xatm/src/atm_comp_nuopc.F90

```diff
@@ -239,6 +239,7 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
        call fld_list_add(fldsFrAtm_num, fldsFrAtm, 'Sa_shum'       )
        call fld_list_add(fldsFrAtm_num, fldsFrAtm, 'Sa_pbot'       )
        call fld_list_add(fldsFrAtm_num, fldsFrAtm, 'Sa_dens'       )
+       call fld_list_add(fldsFrAtm_num, fldsFrAtm, 'Sa_uovern'     )
```

Comment:
I'm not familiar enough with nuopc to know if this will work as is or if you'll need some additional changes, but I'm thinking that e3sm doesn't need nuopc so this can probably be removed.

Reply:
Thanks again! Yes, I was not aware of this. I have updated the file in my latest commit.